rpm/install.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+
+#
+# Determine if search capabilities are enabled
+#
+
+search_enabled="true"
+if [ "${SEARCH_ENABLED}" = "no" ] || [ "${SEARCH_ENABLED}" = "false" ] || [ "${SEARCH_ENABLED}" = "0" ]; then
+    search_enabled="false"
+fi
+echo "---------------- DEBUG: SEARCH"
+echo "Enabled: ${search_enabled}"
+echo "------------------------------"
+
+
+#
+# Configure repository
+#
+
+if [[ -z "${REPOSITORY_CONFIG_FILE}" ]]; then
+  repofile="${__dir}/install-repos.inc"
+else
+  repofile="${REPOSITORY_CONFIG_FILE}"
+fi
+if [ ! -f ${repofile} ]; then
+    echo "File not found: ${repofile}"
+    exit 1
+fi
+echo "---------------- DEBUG: REPO CONFIG"
+cat ${repofile}
+echo "-----------------------------------"
+
+sudo -u root rm -f /etc/yum.repos.d/archivematica.repo
+sudo -u root cp ${repofile} /etc/yum.repos.d/archivematica.repo
+sudo -u root yum update -y
+sudo -u root yum install -y epel-release policycoreutils-python
+
+
+#
+# SELinux tweaks
+#
+
+sudo semanage port -m -t http_port_t -p tcp 8000
```

Comment:
This fails with selinux disabled:
<pre>
+ sudo semanage port -m -t http_port_t -p tcp 8000
SELinux:  Could not downgrade policy file /etc/selinux/targeted/policy/policy.30, searching for an older version.
SELinux:  Could not open policy file <= /etc/selinux/targeted/policy/policy.30:  No such file or directory
/sbin/load_policy:  Can't load policy:  No such file or directory
libsemanage.semanage_reload_policy: load_policy returned error code 2. (No such file or directory).
SELinux:  Could not downgrade policy file /etc/selinux/targeted/policy/policy.30, searching for an older version.
SELinux:  Could not open policy file <= /etc/selinux/targeted/policy/policy.30:  No such file or directory
/sbin/load_policy:  Can't load policy:  No such file or directory
libsemanage.semanage_reload_policy: load_policy returned error code 2. (No such file or directory).
OSError: No such file or directory
[centos@scollazo-centos7 ~]$ getenforce 
Disabled
</pre>

Reply:
Also, on rpms we use ports 81 and 8001, why do we need port 8000 too?

Reply:
> Also, on rpms we use ports 81 and 8001, why do we need port 8000 too?

I just copied/pasted from https://www.archivematica.org/en/docs/archivematica-1.7/admin-manual/installation/installation/#install-centos-redhat-packages.

Reply:
I've filed a bug! https://github.com/artefactual/archivematica-docs/issues/102